content/raw/chat/activity_center.md

```diff
@@ -0,0 +1,116 @@
+    * mention in public chat
+    * mention in a group
+    * mentions in a 1:1
+    * reply to a message from the user
+    * identity check request
+    * community invitation
+* Clicking the "Mark all as read" MUST mark all activity items read of the selected category
+* The activity items MUST be grouped by day sections and sorted by most recent
+* The day sections and their items MUST displayed according to the following criteria
+    * The section with items from today MUST be displayed as "Today"
+    * The section with items from yesterday MUST be displayed as "Yesterday"
+    * The section with items from this year MUST be displayed in the format "Weekday, Day Month" section example: "Mon, 21 April"
+    * The section with items from previous years MUST be displayed in the format "Day Month Year" section example: "Mon, 21 April 2020"
+* Clicking the context menu MUST display the following options:
+    * "Show read notifications", if selected before it MUST toogle to "Hide read notifications"
+    * "Notification settings", if Click it MUST open the notifications settings in profile section
+* The Activity Center MUST display a badge with the total number of currently unread activity items
+* If activity items are are marked as read, the activity center badge MUST be updated accordingly
+
+### Contact Requests
+
+**general**
+* A contact request it MUST appear in the "All" and "Contact Requests" sections.
+
+A contact request item MUST BE EITHER one initiated by the user OR one received by user.
+
+**contact requests initiated by user:**
+* If the user sends a contact request and it has not be accepted or rejected yet, then it MUST appear with the state "Sent"
+* If the user sends a contact request and it has been accepted, then it MUST appear with the state "Accepted"
+* If the user sends a contact request and it has been declined, then it MUST appear with the state "Declined"
+
+**contact requests received by user:**
+* The user MAY accept or reject a contact request in the activity center
+* The user MUST be able to accept a contact request directly in this type of activity item
+* The user MUST be able to decline & block a contact request directly in this type of activity item
+* Clicking the context menu of this type of activity item MUST show 2 options: "view profile" and "Decline and Block"
+* If a contact request item is rejected it MUST display "Declined"
+* If a contact request item is rejected and blocked it MUST display "Declined & Blocked"
+* If a contact request item is accepted it MUST display "Accepted"
+* Clicking on this type of activity item MUST open that chat and jump to that message
+
+### Mentions
+
+* If the user is mentioned the message containing the message MUST appear in the "All" and "Mentions" sections.
+* The user MUST be able to mark this type of item as read
+* Clicking the context menu of this type of activity item MUST show 2 options: "view profile" and "Decline and Block"
+
```

Comment:
```suggestion
* Clicking on this type of activity item MUST open that chat and jump to that message

```

Reply:
+1 on this suggested change, well spotted Richard